src/components/projectSelect/index.tsx

```diff
@@ -104,8 +104,10 @@ export const ProjectSelect: React.VFC<ProjectSelectProps> = ({
           projects={projectGroup.projects}
           name={projectGroup.groupDisplayName}
           onClick={onClick}
-          repoIdentifier={projectGroup?.repo?.id}
-          canClickOnRepoGroup={isProjectSettingsPage && projectGroup?.repo?.id}
```

Comment:
It seems like a more straightforward approach to include the `projectGroup.repo` in the structure returned on line 164. In principle it seems like we should be able to show a repo group that has no projects (even though that's unlikely to happen in practice with our current design -- but you never know!)

Reply:
done